app.js

```diff
@@ -28,6 +33,20 @@ app.use((err, request, response, next) => {
     } next(err);
 });
```

Comment:
```
app.use((err, req, res, next) => {
  if (err.status === 404) {
    res.status(err.status).send({ msg: err.msg });
  } next (err)
});
```

This handler is essentially your custom error handler so it should be a little more flexible to handle any errors that fall into it as each msg may be slightly different. Consider checking for a status and msg to be sure it's a custom error and not one thrown by Express or PSQL